backend/node/base.py

```
@@ -494,71 +754,100 @@ async def _run_genvm(
-            self.timing_callback("GENVM_EXECUTION_END")
+        if host_data is None:
+            host_data = {}
+
+        contract_address = Address(self.contract_snapshot.contract_address)
+
+        if "tx_id" not in host_data:
+            host_data["tx_id"] = "0x"
+        if "node_address" not in host_data:
+            host_data["node_address"] = self.address
+
+        logger = self.logger.with_keys({"tx_id": host_data["tx_id"]})
```

Ruff (0.14.6)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)